modules/core/client/app/init.js

```diff
@@ -30,7 +30,9 @@ angular.module(ApplicationConfiguration.applicationModuleName).run(function ($ro
         if (Authentication.user !== undefined && typeof Authentication.user === 'object') {
           $state.go('forbidden');
         } else {
-          $state.go('authentication.signin');
+          $state.go('authentication.signin').then(function () {
```

Comment:
We could also use something like `$state.go('authentication.signin', {warn: 'Please, sign in first.'})` here. Then, print that in alert in the `signin` template. What do you think?